dev-haskell/hackage-security/hackage-security-0.6.0.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.5.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Hackage security library"
+HOMEPAGE="https://github.com/haskell/hackage-security"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="mtl21 +network-uri"
+
+RDEPEND=">=dev-haskell/base16-bytestring-0.1.1:=[profile?] <dev-haskell/base16-bytestring-0.2:=[profile?]
+	>=dev-haskell/base64-bytestring-1.0:=[profile?] <dev-haskell/base64-bytestring-1.2:=[profile?]
+	>=dev-haskell/cryptohash-sha256-0.11:=[profile?] <dev-haskell/cryptohash-sha256-0.12:=[profile?]
+	>=dev-haskell/ed25519-0.0:=[profile?] <dev-haskell/ed25519-0.1:=[profile?]
+	>=dev-haskell/lukko-0.1:=[profile?] <dev-haskell/lukko-0.2:=[profile?]
+	>=dev-haskell/old-locale-1.0:=[profile?] <dev-haskell/old-locale-1.1:=[profile?]
+	>=dev-haskell/parsec-3.1:=[profile?] <dev-haskell/parsec-3.2:=[profile?]
+	>=dev-haskell/tar-0.5:=[profile?] <dev-haskell/tar-0.6:=[profile?]
+	>=dev-haskell/zlib-0.5:=[profile?] <dev-haskell/zlib-0.7:=[profile?]
+	>=dev-lang/ghc-7.8.2:=
+	|| ( ( >=dev-haskell/cabal-1.14:=[profile?] <dev-haskell/cabal-1.26:=[profile?] )
```

Comment:
I'm not sure that slot operators are allowed in `||` blocks, but I'm going off vague memory here. Do you know if this passed `repoman`?

As a _general_ rule, I correct `||` blocks by using the loosest possible bounds. E.g. the above line becomes `>=dev-haskell/cabal-1.14:=[profile?] <dev-haskell/cabal-3.5:=[profile?]`. But it's also the case, and precisely in this case, that this package won't be built by anything less than `cabal-3.2` anyway due to the way we peg the `cabal` version -- and therefore the `cabal-install` version -- to that provided in `ghc`. So you have some freedom here.

But what should be avoided is simply removing the subslot operators altogether, to allow the `||` blocks. That is undesirable due to the need to rebuild reverse dependencies whenever a dependency is bumped, due to ABI incompatibilities.